package.json

```diff
@@ -36,7 +36,7 @@
     "@angular/platform-browser-dynamic": "~7.0.0",
     "@angular/router": "~7.0.0",
     "@tangoe/goponents": "file:dist/go-lib/tangoe-goponents-0.5.6.tgz",
-    "@tangoe/gosheets": "^0.6.1",
+    "@tangoe/gosheets": "^0.7.1",
```

Comment:
Should we make this version`^0.7.0`?

Reply:
Nope, [gosheets v0.7.1](https://github.com/mobi/gosheets/releases/tag/v0.7.1) is the latest version and includes a hotfix to one of the gosheets mixins.